README.md

```diff
@@ -13,11 +13,13 @@ Client library for accessing Gro Intelligence's agricultural data platform.
 ## Install Gro API client packages
 
 ```sh
-pip install git+https://github.com/gro-intelligence/api-client.git
+pip install groclient
 ```
 
 Note that even if you are using [Anaconda](https://www.anaconda.com/), the API Client install should still be performed using pip and not [conda](https://docs.conda.io/en/latest/).
 
+If you're unable to access PyPI, you can install the latest code from Github: `pip install git+https://github.com/gro-intelligence/api-client.git`
+
```

Comment:
do you think we should mention python2 user here and/or installation.rst?

Reply:
What do you think we should mention? Like, how to install the last Python 2 version?

Perhaps we can wait to see if there are any Python 2 users left before adding documentation for them? WDYT?

Reply:
that should also work